packages/rum-core/src/domain/view/viewMetrics/trackFirstHidden.spec.ts

```diff
@@ -113,6 +113,38 @@ describe('trackFirstHidden', () => {
       } as PerformanceEntry)
 
       firstHidden = trackFirstHidden(configuration)
+      expect(firstHidden.timeStamp).toBe(Infinity as RelativeTime)
```

Comment:
We only need to check the performance observer when we don't have access to events. 

By default, `firstHidden` will only listen to visibility changes and it doesn't check the performance observer.